### ISSUE TITLE
EverydayHabitCalendar not showing correct date

### ISSUE THREAD

**Issue by txwhistler** [mpurses/Sonder#17]
EverydayHabitCalendar is underlining yesterday's date as the current date, which is the date I installed it.  This is in turn throwing off the streak count.  MonthHabitCalendar+5Habits is working as expected.

![EverydayCalendar - 5 Habits](https://user-images.githubusercontent.com/85116026/120199880-4e4ad580-c1e9-11eb-9cf2-523a81377a0f.png)
![EverydayCalendar - Incorrect](https://user-images.githubusercontent.com/85116026/120199881-4e4ad580-c1e9-11eb-9f6c-a19aae91ffbb.png)

**Comment by txwhistler** [comment]
BTW thank you and job well done on this effort!

**Comment by mpurses** [comment]
I ran into this a few times when debugging, I thought I fixed all the cases of it happening but I guess I missed something. If you right-click > Refresh does it fix it?

**Comment by txwhistler** [comment]
No, but I do notice these in the error log when minimizing EveryDayHabitCalendar.ini:

!UpdateMeter: [M5Day151Num] not found (Sonder\EverydayHabitCalendar\StreakNumberMinimized\Minimized.ini)
!HideMeter: [M2Day60Num] not found (Sonder\EverydayHabitCalendar\StreakNumberMinimized\Minimized.ini)

**Comment by mpurses** [comment]
I'll try to get rid of those errors in the next version (it's just running a default script and trying to update the wrong skin) But that doesn't seem to be the issue I think. I'm going to keep trying to reproduce this on my end, but haven't yet.  Can you let me know tomorrow if it is still stuck on the 30th, or if it is constantly one day behind on your computer..

**Comment by mpurses** [comment]
I think I found the issue, still need to test this change over multiple days to make sure it fixes it, but could you give this a try... 

In _Documents\Rainmeter\Skins\Sonder\\@Resources\Calendar_
replace _**Calendar-Measures.inc**_ with the one in this zip file:
[Calendar-Measures.zip](https://github.com/mpurses/Sonder/files/6577674/Calendar-Measures.zip)

**Comment by txwhistler** [comment]
Yes, will do it tonight and let you know the results.


-------- Original message --------
From: Michael Purses ***@***.***>
Date: 6/1/21 11:18 AM (GMT-06:00)
To: mpurses/Sonder ***@***.***>
Cc: txwhistler ***@***.***>, Author ***@***.***>
Subject: Re: [mpurses/Sonder] EverydayHabitCalendar not showing correct date (#17)


I think I found the issue, still need to test this change over multiple days to make sure it fixes it, but could you give this a try...

In Documents\Rainmeter\Skins\Sonder\@Resources\Calendar
replace Calendar-Measures.inc with the one in this zip file:
Calendar-Measures.zip<https://github.com/mpurses/Sonder/files/6577674/Calendar-Measures.zip>

**Comment by txwhistler** [comment]
This seems to have resolved the issue, thanks!  I’ll keep an eye on it over the next few days and will let you know.  Appreciate your time, and am very impressed with your work!


From: Texas ***@***.***>
Sent: Tuesday, June 1, 2021 12:04 PM
To: ***@***.***>; ***@***.***>
Cc: ***@***.***>
Subject: RE: [mpurses/Sonder] EverydayHabitCalendar not showing correct date (#17)

Yes, will do it tonight and let you know the results.


-------- Original message --------
From: Michael Purses ***@***.***>
Date: 6/1/21 11:18 AM (GMT-06:00)
To: mpurses/Sonder ***@***.***>
Cc: txwhistler ***@***.***>, Author ***@***.***>
Subject: Re: [mpurses/Sonder] EverydayHabitCalendar not showing correct date (#17)


I think I found the issue, still need to test this change over multiple days to make sure it fixes it, but could you give this a try...

In Documents\Rainmeter\Skins\Sonder\@Resources\Calendar
replace Calendar-Measures.inc with the one in this zip file:
Calendar-Measures.zip<https://github.com/mpurses/Sonder/files/6577674/Calendar-Measures.zip>

**Comment by mpurses** [comment]
With the change I broke the minimized state from automatically updating colors on a new day, I'll try to fix that over the next few days...

**Comment by mpurses** [comment]
@txwhistler Could you test out this Pre-release, it should have everything fixed. And I want to make sure the upgrade goes well, it should preserve all your existing Variables (custom changes).

https://github.com/mpurses/Sonder/releases/tag/v21.6.0

**Comment by txwhistler** [comment]
Be glad to.
